src/api/routes/application.js

```diff
@@ -11,15 +14,36 @@ export default (app) => {
     /**
      * Creates a new Company Application
      */
-    router.post("/", validators.create, async (req, res, next) => {
-
+    router.post("/", validators.create, applicationMiddleware.exceededCreationTimeLimit, async (req, res, next) => {
```

Comment:
```suggestion
    router.post("/company", validators.create, applicationMiddleware.exceededCreationTimeLimit, async (req, res, next) => {
```

I cannot edit unmodified lines but above, on line 12, it should become:
```js
app.use("/apply", router);
```

This change should _not_ break existing functionality/tests.